docs/docs/behaviors/hold-tap.mdx

```diff
@@ -87,7 +87,7 @@ For example, if you press `&mt LEFT_SHIFT A` and then release it without pressin
 
 If enabled, the hold behavior will immediately be held on hold-tap press, and will release before the behavior is sent in the event the hold-tap resolves into a tap. With most modifiers this will not affect typing, and is useful for using modifiers with the mouse.
 
-:::note Alt/Win/Cmd behavior
+:::info
```

Comment:
```suggestion
:::note[Alt/Win/Cmd behavior]
```
We can keep the note box header using this syntax: https://docusaurus.io/docs/markdown-features/admonitions#specifying-title